backend/src/test/map.rs

```diff
@@ -39,16 +50,36 @@ async fn test_can_find_map() {
     let (token, app) = init_test_app(pool.clone()).await;
 
     let resp = test::TestRequest::get()
-        .uri("/api/maps/-1")
+        .uri("/api/maps")
+        .insert_header((header::AUTHORIZATION, token.clone()))
+        .send_request(&app)
+        .await;
+
+    assert_eq!(resp.status(), StatusCode::OK);
+
+    let result = test::read_body(resp).await;
+    let result_string = std::str::from_utf8(&result).unwrap();
+    let page: Page<MapDto> = serde_json::from_str(result_string).unwrap();
+
+    assert!(page.results.len() == 2);
+
+    let resp = test::TestRequest::get()
+        .uri("/api/maps?name=Other")
```

Comment:
Is this consistent with plant search?

```suggestion
        .uri("/api/maps?search=Other")
```

Reply:
In plants it is also called name.

Reply:
It is also name in plant search

Reply:
Ok, I updated the architecture 9f1eead5099fb86aa7f17b281db39004a697df11